### PR TITLE
Initial implementation of MSP RC stats and info messages

### PR DESCRIPTION
**Pull request by MrD-RC**
This work is to kick off the changes asked for in https://github.com/iNavFlight/inav/issues/10363

Requires configurator https://github.com/iNavFlight/inav-configurator/pull/2230

We are adding two new common MSP V2 messages:
- `MSP2_COMMON_SET_MSP_RC_LINK_STATS` **0x100D**
- `MSP2_COMMON_SET_MSP_RC_INFO` **0x100E**

They are called to update the firmware of changes on the MSP RC receiver/transceiver. `MSP2_COMMON_SET_MSP_RC_LINK_STATS` is sent with every packet and `MSP2_COMMON_SET_MSP_RC_INFO` is sent periodically and when changes are made. Payload size checking has been set so that the payload must at least meet the current defined payload. Future updates to payload would need to be accounted for in updated versions of INAV. But at least, this base data will still be able to be processed.

### MSP2_COMMON_SET_MSP_RC_LINK_STATS
| Parameter         | Size (Bytes) | Datatype | Min  | Max | Units | Notes |
|------------------|---------------|-----------|------|------|-------|--------|
| Sublink ID         | 1                   | uint8      | 0      | 7      |          | Used to distinguish particular antennae or transceiver chips on the module |
| Valid Link          | 1                   | uint8      | 0      | 1      |           | Used to show if the module has a valid link or is in failsafe |
| RSSI %              | 1                   | uint8      | 0      | 100  |          | The RSSI as a 0-100% |
| Uplink RSSI dB | 1                    | uint8     | 0      | 127   | dBm    |  The RSSI in dBm. The value is inverted, so always reads negative or 0. Set to 127 if not used. |
| Downlink LQ    | 1                   | uint8      | 0      | 100   |          | LQ for the downlink |
| Uplink LQ         | 1                   | uint8      | 0      | 100   |          | LQ for the uplink |
| Uplink SNR       | 1                   | int8       | -128 | 127   | dB    | SNR for the uplink |

### MSP2_COMMON_SET_MSP_RC_INFO
| Parameter                | Size (Bytes) | Datatype | Min  | Max | Units | Notes |
|-----------------------|--------------|-----------|------|------|-------|--------|
| Sublink ID                | 1                  | uint8      | 0      | 7      |          | Used to distinguish particular antennae or transceiver chips on the module |
| Uplink TX Power      | 2                  | uint16    | 0      |          | mW  | Uplink power in mW |
| Downlink TX Power | 2                  | uint16    | 0      |          | mW  | Downlink power in mW |
| Band                        | 4                  | char[]     |         |          |          | Band - For example: 868M, 915M, 2.4G, DUAL |
| Mode                       | 6                  | char[]     |         |          |          | Mode - For example: 19Hz, 31Hz, 50Hz. FSK, FLRC, 150Hz, F333Hz, 1000Hz |

### Changes
- Added MSP commands to store the data from the transceiver
- Added missing OSD elements
- Opened up OSD elements that were previously only used by CRSF
- Added downlink LQ and uplink RSSI to the programming framework

### Limitations of current implementation
Due to time constraints. This initial implementation is not a full implementation.
- Currently, only sublink ID 0 will store any data. This will need expanding to allow for up to 8 sublinks.
- Failsafe in the link stats is not used. There is already a failsafe defined in MSP OVERRIDE. We should look at the options and decide the best route to take with this.
- [x] There are no current OSD elements (I will try to add these before the 15th of November deadline) for:
    - [x] Downlink LQ
    - [x] Downlink TX Power
    - [x] Band
    - [x] Mode
- [x] RSSI %, Uplink RSSI, Uplink LQ, Uplink SNR, and Uplink TX Power work with already existing OSD elements. Some are only active when CRSF is selected. So a change will need to be made to open these up for other systems.

### CODE REVIEW

**Comment by b14ckyy** [comment]
Happy on my end. All elements work. Logic triggers also tested and work as intended. Digital fonts will need an Update but if not updated, just 2 icons missing. String text for Mode and Band is clear and up to the RC Link Developer. 

image
image
image
image

File: src/main/drivers/osd_symbols.h [hunk 1]
```diff
@@ -234,6 +234,8 @@
 #define SYM_AH_CH_CENTER            0x166 // 358 Crossair center
 #define SYM_FLIGHT_DIST_REMAINING   0x167 // 359 Flight distance reminaing
 #define SYM_ODOMETER                0x168 // 360 Odometer
+#define SYM_RX_BAND                 0x169 // 361 RX Band
```
Comment (mmosca) [inline src/main/drivers/osd_symbols.h:237]: New symbols? Just in time. ;)

Comment (b14ckyy) [reply src/main/drivers/osd_symbols.h:237]: Shannon is also informed and will add them to his HD font sets.

Comment (MrD-RC) [reply src/main/drivers/osd_symbols.h:237]: Got to keep those DJI folks on their toes. Shannon has added the symbols to his cool fonts.

**Pull request merged**